runtime/queries/metricsview_aggregation.go

```diff
@@ -317,3 +323,16 @@ func (q *MetricsViewAggregation) rewriteToMetricsViewQuery(export bool) (*metric
 
 	return qry, nil
 }
+
+func metricViewExpression(expr *runtimev1.Expression, sql string) (*metricsview.Expression, error) {
+	if expr != nil && sql != "" {
+		return nil, fmt.Errorf("both expression and sql are set")
+	}
+	if expr != nil {
+		return metricsview.NewExpressionFromProto(expr), nil
+	}
+	if sql != "" {
+		return metricssqlparser.ParseSQLFilter(sql)
+	}
```

Comment:
For consistency, can we also add a `metricsview.NewExpressionFromSQL` function that proxies to the `metricssqlparser` internally?

Reply:
This is leading to an import cycle since `metricssqlparser` imports and generates a `metricsview.Query`.